llvm/lib/CodeGen/BranchFolding.cpp

```diff
@@ -25,6 +25,7 @@
 #include "llvm/Analysis/ProfileSummaryInfo.h"
 #include "llvm/CodeGen/Analysis.h"
 #include "llvm/CodeGen/BranchFoldingPass.h"
+#include "llvm/CodeGen/GlobalISel/MachineIRBuilder.h"
```

Comment:
Use of a GlobalISel class here - the post-ISel choice I think should be `MachineInstrBuilder.h`